sw/host/opentitanlib/src/test_utils/load_sram_program.rs

```diff
@@ -6,20 +6,80 @@ use std::fs;
 use std::path::PathBuf;
 use std::rc::Rc;
 use std::str::FromStr;
+use std::time::Duration;
 
 use anyhow::Result;
 use serde::{Deserialize, Serialize};
 use thiserror::Error;
+use structopt::StructOpt;
 
 use crate::impl_serializable_error;
 use crate::io::jtag::{Jtag, RiscvCsr, RiscvGpr, RiscvReg};
 use crate::util::vmem::Vmem;
+use crate::util::parse_int::ParseInt;
 
 use top_earlgrey::top_earlgrey_memory;
 
+
+/// Command-line parameters.
+#[derive(Debug, StructOpt, Clone)]
+pub struct SramProgramParams {
+    /// Path to the VMEM file to load.
+    #[structopt(long)]
+    pub vmem: PathBuf,
+
+    /// Address where to load the VMEM file.
+    #[structopt(long, parse(try_from_str = ParseInt::from_str))]
+    pub load_addr: u32,
+
+    /// Address of the top of the stack.
+    #[structopt(long, parse(try_from_str = ParseInt::from_str))]
+    pub stack_addr: u32,
```

Comment:
nit, for consistency?
```suggestion
    pub stack_pointer: u32,
```